app/core/models/ocelot/stanzas/ChoiceStanza.scala

```diff
@@ -113,7 +113,22 @@ case class ContainsTest(left: String, right: String) extends ChoiceTest {
   def contains(l: BigDecimal, r: BigDecimal): Boolean = l.toString.toLowerCase().contains(r.toString.toLowerCase())
```

Comment:
I wonder do you really need toLowerCase for BigDecimal or LocalDate

Reply:
possibly not, I noticed that it was something Ocelot was doing before doing the contains test, which makes sense for strings but not decimals and dates with numeric months neither